### DISCUSSION TITLE
I have read somewhere that SuperSlicer development was halted due to lack of time?

### DISCUSSION THREAD

**Discussion by VashBaldeus** [supermerill/SuperSlicer#3748] [General]
Is this true? What is the plan for this project, considering that this one supports something that other slicers do not (Klipper G Code Flavor as an example) while other slicers get new features they do not work as good with Klipper as this one does.

Would be nice to have an answer whether this project will resume development sometime in the future?

**Comment by GAZ082** [comment]
This is a non issue. A pre release has been published a couple of weeks ago.

**Reply by vkhodygo** [reply]
This _is_, I'm afraid. No merges, no closed issues, the project is effectively not maintained.

**Reply by cpasoft** [reply]
@GAZ082 , this is not an issue, it is a conversation, and opened as such.
![imagen](https://github.com/supermerill/SuperSlicer/assets/29692062/8e229548-a417-408e-88a0-4c0247fad264)

**Comment by cpasoft** [comment]
Unfortunately, Superslicer seems to have reached a dead end... It was by far the best laminator, but a project of this size is too much for a single programmer, especially if you can't make a living from it. Understandable, and from here I am infinitely grateful to its developer, but unfortunately I think this is the end.

It's time to move on, maybe to Orca, maybe to Prusa...

**Comment by canadank0** [comment]
Support the developer with donations and you don't have to ask these questions

**Reply by rovili** [reply]
I stopped my donations after months of not existing commits.

**Reply by jeremytodd1** [reply]
I'm all for financially supporting development work on open source software like this. I think it's great. I also want my money to go to a project where I think it might actually matter. I'm not going to throw money at someone who hasn't shown much progress in the last year. SS is still missing features from PrusaSlicer 2.5, which is quite old at this point.

I'm grateful for all the work that supermerill has put into SS. I used the software exclusively for a few thousand print hours. If it's at the point where supermerill either isn't motivated to work on it anymore, or simply doesn't have the time due to life happenings, I believe that it'd be a good idea for someone to fork SuperSlicer so it can have more consistent development.

There is a reason that more and more people are looking into other slicers. Slicers are moving very quickly at the moment with some huge features. They don't want to miss out on those and I don't blame them. I've since switched to OrcaSlicer and have absolutely no regrets. It's great being on a slicer that actually gets updates again.

**Reply by SirOink** [reply]
I'm fully willing to financially support a project like this if it were updated more often. pretty sure there's also companies out there who'd sponsor the project (like klipper hunted some down recently, allowed their brand to be used for marketing purposes with new 3d printers and is now secure for the foreseeable future) if he emailed a few 3dp places.

but the main problem here is that supermerill isn't accepting or asking for help to maintain (aka moderate) the projects github and hasn't done so for the 12 years (unless that's a lie and based on slic3r? I've never really figured that part of 'hub out) the project has existed.

I don't know supermerill personally, so it's hard to say WHY he hasn't tried to find people willing to help as maintainers who can review helpful pull requests and isn't solely responsible for keeping things maintained 24/7 in his spare time for nothing but glory.

the usual reasons are not having the motivation or time to vet strangers (likely reason), ego - wanting full control of his baby, or simple capitalism... either failed or future.

the best thing he could do is switch it to a group project thingy thing (Idk what it's called on github, but it's so the community can kick out inactive owners/those whom act nefariously) and sit back as the projects founder, only make tweaks in his spare time and not worry about us sad cunts that would kill for an update

**Comment by canadank0** [comment]
> I'm all for financially supporting development work on open source software like this. I think it's great. I also want my money to go to a project where I think it might actually matter. I'm not going to throw money at someone who hasn't shown much progress in the last year. SS is still missing features from PrusaSlicer 2.5, which is quite old at this point.
> 
> I'm grateful for all the work that supermerill has put into SS. I used the software exclusively for a few thousand print hours. If it's at the point where supermerill either isn't motivated to work on it anymore, or simply doesn't have the time due to life happenings, I believe that it'd be a good idea for someone to fork SuperSlicer so it can have more consistent development.
> 
> There is a reason that more and more people are looking into other slicers. Slicers are moving very quickly at the moment with some huge features. They don't want to miss out on those and I don't blame them. I've since switched to OrcaSlicer and have absolutely no regrets. It's great being on a slicer that actually gets updates again.

for me OrcaSlicer is same like Cura. many cool features, but hard way to tweak the basics. I want to be able to manage my printing process in detail and not get automated solution. So far SuperSlicer is still the hardcore tool let you do that. Otherwise i will switch to PrusaSlicer.

**Reply by jeremytodd1** [reply]
Out of curiosity, can you give a couple examples of what you mean?

**Reply by canadank0** [reply]
Extrusion width settings and spacing in Width and flow, full speed area in Print settings. Infill settings for connected anchors - etc.

**Reply by jeremytodd1** [reply]
Hm I think the only thing that Orca is missing from those is line spacing. 

- Extrusion width settings: You can set them to number values or percentage based numbers. Currently the percentages are based off of layer heights, which makes no sense, but the next release will be changing this to percentages based off of the nozzle size, just like how SuperSlicer handles it. This feature is already in the latest nightly so you can use this already if you want.
- Line spacing is not in Orca, correct. I haven't dealt with this too much my self but can definitely see the appeal of it. 
- Orca has plenty of speed settings. If anything I think I'd say that Orca has more settings for cooling than SuperSlicer currently. An example is dynamic overhang speeds. Any extra clarity on this point would be appreciated incase I missed something.
- The last version of Orca did bring in settings for infill anchors.
